database/migrations/2020_09_17_000000_change_default_membership_type_id_in_users_table.php

```diff
@@ -16,7 +16,8 @@ public function up()
         Schema::table('users', function (Blueprint $table) {
             // Changing to smallint because doctrine crashes when trying to change the tinyint column
             // https://github.com/laravel/framework/issues/8840
-            $table->smallInteger('membership_type_id')->unsigned()->default(2)->change(); // default to free membership as demo will be deprecated
+            // workaround for this is to convert SMALLINT first, then TINYINT
+            $table->smallInteger('membership_type_id')->tinyInteger('membership_type_id')->unsigned()->default(2)->change(); // default to free membership as demo will be deprecated
```

Comment:
smallInteger and tinyInteger?

Reply:
Yes, this is known issue. GIthub link is provided, if we use only one smallintergar or tinyInteger then doctrine crashes. So this is work-around.